src/nestdoor/settings.py

```diff
@@ -30,10 +28,12 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 # Edited for Digital Ocean Deployment
-DEBUG = os.getenv("DEBUG", "False") == "True"
+DEBUG = os.getenv("DEBUG", "True") == "True"
 
 # Edited for Digital Ocean Deployment
-ALLOWED_HOSTS = os.getenv("DJANGO_ALLOWED_HOSTS", "nestdoor-app-zcquw.ondigitalocean.app", "127.0.0.1,localhost").split(",")
```

Comment:
Pls review this line.

Reply:
What do you mean by review this line?  Are you referring to the ALLOWED_HOSTS?